spring-cloud-dataflow-admin-local/pom.xml

```diff
@@ -27,6 +27,12 @@
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-web</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.hsqldb</groupId>
+			<artifactId>hsqldb</artifactId>
+			<scope>test</scope>
+			<optional>true</optional>
```

Comment:
Not sure I understand the aim of test + optional here. What are we trying to achieve? test dependencies are not brought transitively anyway